content/2. Ondersteunende-informatie/3. Processen/3. Ontwerpproces/1. Opstellen ontwerp/Schematechnieken/1. Klassendiagram.md

```diff
@@ -0,0 +1,163 @@
+```mermaid
+classDiagram
+
+class Hotel {
+    naam
+    locatie
+}
+
+
+class Kamer {
+	kamernummer
+	aantal bedden
+	prijs per nacht
+}
+
+Hotel "1" -- "*" Kamer : heeft
+```
+*Diagram 2: een eenvoudig voorbeeld van een klassendiagram. In dit geval een domeinmodel. *
+
+
+## Hoe gebruik je het klassendiagram
+Het klassendiagram kun je op verschillende manieren in je development cyclus gebruiken.
+
+| Toepassing   | Toelichting                                                                                                                                                                                                                                                                                                                                  |
+| ------------ | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Brainstorm   | Het klassendiagram maak je wellicht veelal digitaal maar de `taal` is veel breder toepasbaar. Als je met je collega's zoekt naar de best passende oplossing voor een probleem is de syntax uitstekend geschikt om op een whiteboard toe te passen en zo ideeën uit te wisselen.                                                              |
+| Analyse      | Bij het onderzoeken van je probleemdomein doe je allerlei inzichten op. Bijvoorbeeld welke informatie een rol speelt bij het probleemdomein. Het klassendiagram is dan een effectieve methode om dit vast te leggen.<br>Dit zelfde model kun je gebruiken om je ideeën aan te scherpen of bij te stellen om zo tot een beter model te komen. |
+| Afstemming   | Het klassendiagram is een effectief middel om af te stemmen tussen stakeholders. Een eenvoudig model(zoals het domeinmodel) gebruik je in gesprek met je klant. Complexere modellen gebruik je om af te stemmen met je collega developers.                                                                                                   |
+| Realisatie   | Doordat je al hebt nagedacht over je ontwerp ga je makkelijker de realisatie door. Het klassendiagram maakt het overdraagbaar en geeft inzicht. Tijdens de realisatie krijg je wellicht nieuwe inzichten. Gebruik het model om je ideeën aan te scherpen en af te stemmen.                                                                   |
+| Documentatie | Door het klassendiagram op te nemen in je functionele (domeinmodel) en technische (technisch klassendiagram) documentatie biedt je snel inzicht in het domein en de werking van de applicatie. Zo hoef je niet eerst vele regels code te doorlopen om ongeveer te weten waar het over gaat.                                                  |
+
+## Hoe maak je het klassendiagram
```

Comment:
Dit kopje bevat eigenlijk de informatie voor het kopje "Hoe zit het klassendiagram in elkaar?". 

Verder is het handig voor de consistentie om alleen de drie standaard vragen als ## kopjes te houden en andere eigen kopjes onder deze drie te zetten.